app/views/shared/_header.html.erb

```diff
@@ -30,7 +30,8 @@
 							<% if current_user.image.blank? %>
 								<span class="avatar"><%= current_user.name.first %></span>
 							<% else %>
-								<span class="avatar" style="background-image: url(<%= current_user.image %>)"></span>
+								<span id="user-avatar" class="avatar" style="display: none;"><%= current_user.name.first %></span>
```

Comment:
If you are rendering the span anways in both cases, get rid of the else by inverting the if and only add the next segment in that case.

Reply:
In the else statement, the avatar is defaulted to hidden, which prevents a tiny visual glitch every time a page is loaded. If removed, every time the page is loaded, the default avatar and the user image will both appear side by side for a split second, and then 1 will be hidden